lib/MTT/Test/Build/Noop.pm

```diff
@@ -12,13 +12,24 @@ package MTT::Test::Build::Noop;
 
 use strict;
 
+use MTT::Values;
+
 #--------------------------------------------------------------------------
 
 sub Build {
+    my ($ini, $mpi_install, $config) = @_;
+
+    # In case of tests located in different folder and no much space available
+    # on target scratch, it can be useful to point another location of prebuilt
+    # tests.
+    my $test_bin_dir = Value($ini, $config->{full_section_name}, "test_bin_dir");
+
     my $ret;
     $ret->{test_result} = MTT::Values::PASS;
     $ret->{exit_status} = 0;
     $ret->{result_message} = "Success";
+    $ret->{bindir} = $test_bin_dir if $test_bin_dir;
```

Comment:
We want `$ret->{bindir}` to exist but set to `undef` if it wasn't set, right?

I.e., the infrastructure should always be able to assume that the `bindir` (or `installdir`, per the previous comment) field *exists*, even if it's effectively empty.

Reply:
Not necessarily. I added a check into Run.pm:
https://github.com/open-mpi/mtt/pull/604/files#diff-bd02d29cb58bf48db2504639705bcd7aR433, which set `binder = srcdir` if first one missed. Otherwise I need to add `$ret->{bindir} = under` into every build module for consistency.
Anyway, see next comment.